HelpSource/Classes/FunctionDef.schelp

```diff
@@ -100,6 +102,29 @@ code::
 { |a = 9, b = 10 ... c| a + b }.def.argumentString(asArray: true);
 ::
 
+method::filenameSymbol
+
+Return the file path (as a link::Classes/Symbol::) where the code was defined, if the file has not been saved (or you are using a terminal interface) returns nil.
```

Comment:
–> Return the file path (as a link::Classes/Symbol::) where the function was defined,